shared-bindings/bitmaptools/__init__.c

```diff
@@ -949,6 +949,132 @@ STATIC mp_obj_t bitmaptools_obj_draw_circle(size_t n_args, const mp_obj_t *pos_a
 
 MP_DEFINE_CONST_FUN_OBJ_KW(bitmaptools_draw_circle_obj, 0, bitmaptools_obj_draw_circle);
 
+//|     def blit(
+//|         dest_bitmap: displayio.Bitmap,
+//|         source_bitmap: displayio.Bitmap,
+//|         x: int,
+//|         y: int,
+//|         *,
+//|         x1: int,
+//|         y1: int,
+//|         x2: int,
+//|         y2: int,
+//|         skip_index: int,
```

Comment:
Since the API is changing, be explicit here too.

```suggestion
//|         skip_source_index: int,
```